scripts/docker/debian/stretch/Dockerfile.doc

```diff
@@ -31,12 +30,17 @@ RUN mkdir -p ${GTEST_ROOT} \
     && tar -zxvf gtest.tar.gz --strip-components=1 -C ${GTEST_ROOT} \
     && rm gtest.tar.gz
 
+# ronn
+# Unfortunately the package [`ruby-ronn`](https://packages.debian.org/buster/all/ruby-ronn/filelist) does not include the `ronn` executable.
+RUN gem install ronn
+
 # Create User:Group
 # The id is important as jenkins docker agents use the same id that is running
 # on the slaves to execute containers
 ARG JENKINS_GROUPID
 RUN groupadd \
     -g ${JENKINS_GROUPID} \
+    -f \
```

Comment:
Why is this needed?

Reply:
Otherwise building the image on macOS will not work (see also commit https://github.com/ElektraInitiative/libelektra/pull/2819/commits/8b916927d9b459711813bde3192bee0ba85eff48).